src/tasks/MotionForceTask.h

```diff
@@ -527,6 +565,12 @@ class MotionForceTask : public TemplateTask {
 	 */
 	void initialSetup();
 
+	// singularity handling
+	std::unique_ptr<SingularityHandler> _singularity_handler;
+	MatrixXd _Lambda_s, _Lambda_ns, _Lambda_s_modified, _Lambda_ns_modified;
```

Comment:
As mentioned before, we probably don't need those as they are already in the singularity handler class